sdk/containerregistry/azure-containers-containerregistry/src/test/java/com/azure/containers/containerregistry/ContainerRepositoryAnonymousAccessTests.java

```diff
@@ -4,6 +4,8 @@
 package com.azure.containers.containerregistry;
 
 import com.azure.core.http.HttpClient;
+import com.azure.core.test.implementation.ImplUtils;
```

Comment:
nit: this shouldn't be accessed

Reply:
Yup looks like a bad merge - cleaning it.